test/data/recordings/TestAccAttackProtectionBreachedPasswordDetection.yaml

```diff
@@ -1254,7 +1254,7 @@ interactions:
       trailer: { }
       content_length: -1
       uncompressed: true
-      body: '{"enabled":false,"shields":["admin_notification","user_notification","block"],"admin_notification_frequency":["weekly","monthly","daily","immediately"],"method":"standard","stage":{"pre-user-registration":{"shields":[]}}}'
+      body: '{"enabled":false,"shields":["admin_notification","user_notification","block"],"admin_notification_frequency":["weekly","monthly","daily","immediately"],"method":"standard","stage":{"pre-user-registration":{"shields":["admin_notification"]}}}'
```

Comment:
This is getting updated on the wrong interaction @nialdaly, if you want I can push the regenerated recording.

Reply:
That would be great!

Reply:
Alright, one moment ⌛